### PR TITLE
PANGOLIN-2332:  add image and review models

### PR DESCRIPTION
**Pull request by valoriecarli**
Includes the following models:

`Image `
-- 2 bonus presets

`Review`(with draft)

### CODE REVIEW

File: models/image/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 commercetools GmbH
```
Comment (tylermorrisford) [inline models/image/LICENSE:3]:
```suggestion
Copyright (c) commercetools GmbH
```

File: models/review/src/generator.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { ClientLogging } from '@commercetools-test-data/commons';
+import { sequence, fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TReview } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+// https://docs.commercetools.com/api/projects/reviews#reviewdraft
```
Comment (tylermorrisford) [inline models/review/src/generator.ts:8]: Should this be:
```suggestion
// https://docs.commercetools.com/api/projects/reviews#review
```
?

File: models/tax-category/src/generator.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ const generator = Generator<TTaxCategory>({
     key: fake((f) => f.lorem.slug(2)),
     name: fake((f) => f.lorem.words(2)),
     description: fake((f) => f.lorem.sentence()),
-    rates: fake(() => [random()]),
+    rates: fake(() => [TaxRate.random()]),
```
Comment (tylermorrisford) [inline models/tax-category/src/generator.ts:18]: 💪 💪 💪

**Review by tylermorrisford — APPROVED** [review]

File: models/tax-category/src/generator.ts (hunk 3 above)
Comment (tylermorrisford) [reply models/tax-category/src/generator.ts:18]: 😆

File: models/image/src/presets/commercetools-pos-aem.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import Image from '../builder';
+
+const commercetoolsPosAem = () =>
```
Comment (jaikamat) [inline models/image/src/presets/commercetools-pos-aem.ts:3]: What're POS and AEM?

File: models/review/LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 commercetools GmbH
```
Comment (jaikamat) [inline models/review/LICENSE:3]:
```suggestion
Copyright (c) commercetools GmbH
```

File: models/review/src/generator.ts [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+const generator = Generator<TReview>({
+  fields: {
+    id: fake((f) => f.datatype.uuid()),
+    version: sequence(),
+    key: fake((f) => f.lorem.slug(2)),
+    uniquenessValue: null,
+    locale: fake((f) => f.random.locale()),
+    authorName: fake((f) => f.name.fullName()),
+    title: fake((f) => f.word.interjection()),
+    text: fake((f) => f.hacker.phrase()),
```
Comment (jaikamat) [inline models/review/src/generator.ts:19]: Lol, nice touch

**Review by jaikamat — APPROVED** [review]

**Pull request merged**